### ISSUE TITLE
Incorrect parsing of metadata

### ISSUE THREAD

**Issue by j0dan** [NilsRo/OctoPrint-SlicerEstimator#27]
I have this in my gcode:
```
;Slicer info:filament_settings_id;Prusament ASA
;Slicer info:print_settings_id;0.2mm SPEED (dan)
```

But Slicer Estimator parses it as:
![image](https://user-images.githubusercontent.com/10273471/150213062-fbd40776-3a4b-499a-9d26-3e2fc9a5cc89.png)
```
ament_settings_id
print_settings_id
```

**Comment by NilsRo** [comment]
Please switch to development channel in the plugins settings, update from there and report back. I pushed a fix that should correct this.

**Comment by j0dan** [comment]
Same problem after switching to channel and updating.

Thanks for looking at it.

**Comment by NilsRo** [comment]
Did you uploaded the file again? Overwriting it is fine. The metadata is read only on upload where I found a bug in the string split.

**Comment by j0dan** [comment]
Your release just now for 1.2.2 fixed it.

**Comment by NilsRo** [comment]
Mh, that is strange as I corrected it only in the upload routine and the version change comes next for preparing the release...

But very fine if it works now. Could you anyhow upload the octoprint.log. Perhaps I can see something that is going not as expected is it should not heal by itself without uploading. :-)   And you can switch back to normal release channel. I published version 1.2.2 also there.
